source/features/bugs-tab.tsx

```diff
@@ -11,6 +11,13 @@ import {getRepo} from '../github-helpers';
 import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 
+interface LabelInfo {
```

Comment:
Please stop typing API responses. This is another example of why it should not be done: it's wrong.